src/helpers/hsla.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+
+import hslToHex from '../internalHelpers/_hslToHex'
+import hslToRgb from '../internalHelpers/_hslToRgb'
+
+type HslColor = {
+  hue: number,
+  saturation: number,
+  lightness: number,
+  alpha: number,
+}
+
+/**
+ * Returns a string value for the color. The returned result is the smallest possible hex notation.
```

Comment:
This doesn't make any sense, "is the smallest possible hex notation"? It's actually converted to `rgba` with a hex fallback, right? Hex notation can't have transparency...